tests/test_handler.py

```diff
@@ -430,3 +430,12 @@ def test_log_request_truncate_negative(enabled):
             settings=None,
             crawler=crawler,
         )
+
+
+@ensureDeferred
+async def test_addon(mockserver):
```

Comment:
This test needs skipping on old Scrapy and I think this is the only place that check is needed.